src/ray/raylet_client/raylet_client.h

```diff
@@ -139,10 +139,22 @@ class DependencyWaiterInterface {
   virtual ~DependencyWaiterInterface(){};
 };
 
+/// Interface for fetching config.
+class ConfigInterface {
+ public:
+  /// Get the system config from Raylet.
+  /// \param callback Callback that will be called after raylet replied the system config.
+  virtual void GetSystemConfig(
+      const rpc::ClientCallback<rpc::GetSystemConfigReply> &callback) = 0;
+
+  virtual ~ConfigInterface(){};
+};
+
```

Comment:
Can we just add this to the `RayletClientInterface` instead of adding a new one?

Reply:
Maybe we can fetch the job config here as well.

Reply:
Unless there's a concrete reason to have the interface right now (e.g., to support unit testing) let's omit it, excessive interfaces make making changes more difficult

Reply:
Ah, I didn't know these interfaces are for unit testing. Will update it.